.github/workflows/test.yml

```diff
@@ -13,8 +13,4 @@ jobs:
       uses: actions/checkout@v2
     - name: Install modules.
       run: yarn
-    - name: Test and collect coverage.
-      uses: artiomtr/jest-coverage-report-action@v1.3
-      with:
-        github_token: ${{ secrets.GITHUB_TOKEN }}
-        test_script: yarn test --coverage
+    - run: yarn test:ci
```

Comment:
This looks kinda out of scope for this PR.

Reply:
Yeja but without this change the tests weren't passing.

Reply:
Well should be fixed after #400.

Reply:
Yeah but it weren't and I was trying to find the issue 😅
Anyway, I don't think will conflict.

Reply:
Anyways, its safe to revert, #400 will fix this.